### PR TITLE
Check for duplicates in examples table added

### PR DESCRIPTION
**Pull request by xsmrcek**
Should solve issue #291 .

### CODE REVIEW

File: jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/ExamplesTableUtil.java [hunk 1]
```diff
@@ -164,6 +164,56 @@ public static void assertMandatoryColumns(ExamplesTable examplesTable, String ..
...
+            for (int row = 0; row < examplesTable.getRowCount(); row++) {
+                if (duplicates.contains(examplesTable.getRow(row).get(column))){
+                    continue;
+                }
+                StringBuilder sb = new StringBuilder();
+                checkColumnForDuplicates(row, sb, duplicates, examplesTable, column);
+                if (!sb.toString().equals("")){
+                    softly.fail(sb.toString() + "}");
+                }
+            }
```
Comment (xsmrcek) [inline jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/ExamplesTableUtil.java]: return sb, do not send it in.

Comment (xsmrcek) [reply jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/ExamplesTableUtil.java]: Send softly instead, and fail in method.

Comment (xsmrcek) [reply jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/ExamplesTableUtil.java]: recursion?!

**Comment by paveljandejsek** [comment]
My 2c - previous version seemed much more readable, this while more functional and "pretty" makes less sense to me and takes me more time to understand 🤕

**Pull request merged**